cumulusci/tasks/push/tasks.py

```diff
@@ -229,15 +235,30 @@ def _init_task(self):
     def _init_options(self, kwargs):
         super(SchedulePushOrgList, self)._init_options(kwargs)
 
+        neither_file_option = "orgs" not in self.options and "csv" not in self.options
+        both_file_options = "orgs" in self.options and "csv" in self.options
+        if neither_file_option or both_file_options:
+            raise TaskOptionsError(
+                "Please call this task with either the `orgs` or `csv` option."
+            )
         # Set the namespace option to the value from cumulusci.yml if not
         # already set
         if "namespace" not in self.options:
             self.options["namespace"] = self.project_config.project__package__namespace
         if "batch_size" not in self.options:
             self.options["batch_size"] = 200
+        if "csv" not in self.options and "csv_field_name" in self.options:
+            raise TaskOptionsError("Please provide a csv file for this task to run.")
+        if "csv" in self.options and "csv_field_name" not in self.options:
+            self.options["csv_field_name"] = "OrganizationId"
```

Comment:
One last thing:

Since you're also checking whether `csv_field_name` is set in `_get_orgs()` you could dispense with this check by using `self.options.get`:
```python
if "csv" in self.options:
    with open(self.options.get("csv")) as csvfile:
        reader = csv.DictReader(csvfile)
        return [row[self.options.get("csv_field_name", "OrganizationId")] for row in reader]
```